demo/router-demo.js

```diff
@@ -0,0 +1,195 @@
+import { LitElement } from '@dreamworld/pwa-helpers/lit-element';
+import { html, css } from 'lit-element';
+// import { init, registerFallbackCallback} from '../index.js';
+import * as router from '../index.js';
+
+import { store } from './redux/store.js';
+import { connect } from 'pwa-helpers/connect-mixin.js';
+
+import "@material/mwc-tab-bar";
+import "@dreamworld/dw-button";
+import "@dreamworld/dw-dialog";
+import "./demo-page-1.js";
+import "./demo-page-2.js";
+import "./demo-page-3.js";
+import "./demo-page-4.js";
+
+const URLs = {
+  pages: [
+    {
+      name: 'page1',
+      pathPattern: '/demo/:companyId/page1',
+      pathParams: {
+        companyId: Number
+      }
+    },
+    {
+      name: 'page2',
+      pathPattern: '/demo/:companyId/page2',
+      pathParams: {
+        companyId: Number
+      }
+    },
+    {
+      name: 'page3',
+      pathPattern: '/demo/:companyId/page3',
+      pathParams: {
+        companyId: Number
+      }
+    },
+    {
+      name: 'page4',
+      pathPattern: '/demo/:companyId/page4',
+      pathParams: {
+        companyId: Number
+      }
+    }
+  ],
+  dialogs: [
+    {
+      name: 'dialog1',
+      pathPattern: '#dialog1',
+    },
+    {
+      name: 'dialog2',
+      pathPattern: '#dialog2',
+    }
+  ]
+}
+
+class routerDemo extends connect(store)(LitElement) {
+  static get styles() {
+    return [
+      css`
+        dw-button {
+          margin-top: 24px;
+          margin-bottom: 24px;
+        }
+
+        dw-dialog dw-button {
```

Comment:
if you don't need same css for the all the dw-button than you should not write css for all dw-button. you can add class based css or parent's child base css.